### ISSUE TITLE
[$10][MSFT-152] [Challenge Submission Page]  'Back to challenge' button contrast between foreground and background colors won't meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2842]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges/30095032/submit and Login using valid credentials
2. Check the 'Back to challenge' button

**Expected Result**
Expected contrast ratio of 4.5:1

- Passes AA, if you change background color to #3A3A3A. New contrast ratio: 4.55
- Passes AA, if you change text color to #737383. New contrast ratio: 4.54

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

**Actual Result**
Element has insufficient color contrast of 2.43 (foreground color: #a3a3ae, background color: #fcfcfc, font size: 13.5pt, font weight: normal)

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F59

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096178 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096178 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by Dara-K** [comment]
@r0hit-gupta Please change the color to #6b6b6b

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096178 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2842 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097136 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097136 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
URL : https://test-community-app.topcoder-dev.com/challenges/30055274/submit

**Comment by tosha5252** [comment]
Color still #a3a3ae...verified on Chrome, Edge and with WAVE tool

Pls change to #6b6b6b

**Comment by nithyaasworld** [comment]
Make sure not to hard-code any colours unless and until directed. Use variables always.

**Comment by lakshmiathreya** [comment]
@Sushil I verified on multiple browsers in Mac on Prod - the color is #a3a3ae. 
<img width="1651" alt="Screenshot 2019-07-29 at 10 16 53 PM" src="https://user-images.githubusercontent.com/51733042/62066391-d1e2b680-b24e-11e9-9874-c36dae8ffd08.png">

**Comment by sushilshinde** [comment]
@crazyk07 @nithyaasworld @crazyk07 this another case where code is present on the production 

https://github.com/topcoder-platform/community-app/pull/2931/files

https://github.com/topcoder-platform/topcoder-react-ui-kit/blob/fdfe7fd7cff05490dd87abe3f44542ec28fbbe0f/src/styles/_mixins/variables.scss#L28

https://github.com/topcoder-platform/community-app/blob/894e4486f05201b26ffa041cb1d8e5d502eb07f9/src/shared/components/SubmissionPage/FilestackFilePicker/styles.scss#L27

**Comment by sushilshinde** [comment]
Fixing will be available in an hour on production

**Comment by lakshmiathreya** [comment]
Verified color set to #6b6b6b.
Chrome on MacOS/Win 10.

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097136<br/><br/>```This is an automated message for crazyk via Topcoder X```
